apps/site/components/Common/Tooltip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import type { ComponentProps, FC, PropsWithChildren, ReactNode } from 'react';
+
+import styles from './index.module.css';
+
+type TooltipProps = {
+  asChild?: boolean;
+  content: ReactNode;
+  kind?: 'default' | 'warning' | 'error';
+  side?: ComponentProps<typeof TooltipPrimitive.Content>['side'];
+};
+
+const Tooltip: FC<PropsWithChildren<TooltipProps>> = ({
+  kind = 'default',
+  children,
+  content,
+  asChild = false,
+  side = 'bottom',
+}) => (
+  <TooltipPrimitive.Provider delayDuration={200}>
+    <TooltipPrimitive.Root>
+      <TooltipPrimitive.Trigger asChild={asChild}>
+        {children}
+      </TooltipPrimitive.Trigger>
+      <TooltipPrimitive.Portal>
+        <TooltipPrimitive.Content
+          side={side}
+          sideOffset={4}
+          className={`${styles[kind]} ${styles.content}`}
```

Comment:
may be use `classname` is used on other components

Reply:
AFAIK, we are only using `classNames` if we are going to add a conditional class 🤔

Reply:
Oh, I didn't know. Personally I always use `classnames` when there are several `class` to avoid having a string with undifined in it.